.circleci/config.yml

```diff
@@ -169,6 +169,73 @@ jobs:
           command: pytest -v -m integration_test
           env:
             - PREFECT__LOOP_SERVICE__DEBUG_MODE: true
+            
+  publish_docker_images:
+    machine:
+      image: ubuntu-2204:2022.04.1
+    environment:
+      DOCKER_BUILDKIT: 1
+      BUILDX_PLATFORMS: linux/amd64,linux/arm64
+    steps:
+      - run:  # https://circleci.com/blog/building-docker-images-for-multiple-os-architectures/
+          name: Install buildx
+          command: |
+            BUILDX_BINARY_URL="https://github.com/docker/buildx/releases/download/v0.8.2/buildx-v0.8.2.linux-amd64"
+            curl --output docker-buildx \
+              --silent --show-error --location --fail --retry 3 \
+              "$BUILDX_BINARY_URL"
+            mkdir -p ~/.docker/cli-plugins
+            mv docker-buildx ~/.docker/cli-plugins/
+            chmod a+x ~/.docker/cli-plugins/docker-buildx
+            docker buildx install
+            # Run binfmt
+            docker run --rm --privileged tonistiigi/binfmt:latest --install "$BUILDX_PLATFORMS"
+          filters:
+            branches:
+              only: master
+            tags:
+              only: /(2\d{3}\.(0[1-9]|1[0-2])\.(0[1-9]|[12]\d|3[01])).*/
```

Comment:
is this an `OR` (couldn't find docs on this behaviour)? these steps should run either on master, or on tags (it's a combination of the filters in the later steps). if branches,tags acts like an `AND`, it needs fixing (also in the login step below).

Reply:
hmm this needs fixing. the steps still run and fails on missing docker hub env vars 🤔 does `branches` specify the _target_ branch?